descriptors/ModuleDescriptor-template.json

```diff
@@ -668,6 +679,9 @@
         "inventory-storage.holdings-types.item.post",
         "inventory-storage.holdings-types.item.put",
         "inventory-storage.holdings-types.item.delete",
+        "inventory-storage.holdings-sources.item.post",
+        "inventory-storage.holdings-sources.item.put",
+        "inventory-storage.holdings-sources.item.delete",
```

Comment:
All these 7 permissions were in file system-user-permissions but were missed in ModuleSecriptor-template